tests/unit_tests/test_suassystem_sda_converter.py

```diff
@@ -50,15 +148,6 @@ def test_add_obstacle(self):
         self.assertEqual(obstacle_map.obstacles.size, 1)
 
     def test_reset_obstacles(self):
-
-        #Test the reset_obstacles() method
-
-        new_obstacle = interop.StationaryObstacle(38.8703041, -77.3214035, 0, 60.950000762939453)
-        self.sda_converter.add_obstacle(Location(38.8703041, -77.3214035, 60.950000762939453), new_obstacle)
-        obstacle_map = self.sda_converter.obstacle_map
-
-        self.assertEqual(obstacle_map.obstacles.size, 1)
```

Comment:
You need an obstacle in the map before you test reset obstacles. That way, you ensure that the reset obstacles method works correctly.

Reply:
This comment still stands. Please fix.